imgui_test_suite/imgui_tests_widgets_inputtext.cpp

```diff
@@ -679,33 +679,40 @@ void RegisterTests_WidgetsInputText(ImGuiTestEngine* e)
                 KeyPressAndDebugPrint(chord_word_prev);
                 IM_CHECK_EQ(state->GetCursorPos(), 6 + 5 + 2 + 3);          //  "Hello world. Foo|.bar!!!" // VS-Win: STOP, GitHubWeb-Win: SKIP
 
-                KeyPressAndDebugPrint(chord_word_prev;
+                KeyPressAndDebugPrint(chord_word_prev);
 #endif
                 IM_CHECK_EQ(state->GetCursorPos(), 6 + 5 + 2);              //  "Hello world. |Foo.bar!!!"
 
                 KeyPressAndDebugPrint(chord_word_prev);
-#if IMGUI_BROKEN_TESTS // may be fixed by #6067
+#if IMGUI_VERSION_NUM >= 18930
                 IM_CHECK_EQ(state->GetCursorPos(), 6 + 5);                  //  "Hello world|. Foo.bar!!!" // VS-Win: STOP, GitHubWeb-Win: SKIP
                 KeyPressAndDebugPrint(chord_word_prev);
 #endif
                 IM_CHECK_EQ(state->GetCursorPos(), 6);                      //  "Hello |world. Foo.bar!!!"
 
                 KeyPressAndDebugPrint(chord_word_prev);
                 IM_CHECK_EQ(state->GetCursorPos(), 0);                      // "|Hello world. Foo.bar!!!"
+
+                KeyPressAndDebugPrint(chord_word_prev);
+                IM_CHECK_EQ(state->GetCursorPos(), 0);                      // "|Hello world. Foo.bar!!!" // (no-op)
```

Comment:
Not sure if this functionality is tested elsewhere and this test is useless or not.